apis/form/formapi.yaml

```diff
@@ -1,40 +1,48 @@
 openapi: 3.0.1
 info:
   title: Form API(s)
-  description: |-
-    - The Form APIs allow you to create and update metadata form fields based on type, subtype and action. Use the APIs to configure form fields as required for the specific form.
+  description: >-
+    - The Form APIs allow you to create and update metadata form fields based on
+    type, subtype and action. Use the APIs to configure form fields as required
+    for the specific form.
+
     - The URL for Form API is `/data/v1/form/`.
-  version: "1.0"
+  version: '1.0'
 servers:
-- url: http://staging.open-sunbird.org/api/data/v1/form
+  - url: 'http://staging.open-sunbird.org/api/data/v1/form'
```

Comment:
Please add the url as `https://staging.open-sunbird.org/api` only and add the rest of the part in the api path below.

Reply:
ok